src/components/TableBody.js

```diff
@@ -11,12 +14,21 @@ class TableBody extends Component {
     var lockstate = this.props.lockstate || 'unlocked';
     var total_working_hours = this.props.totaldays * 8;
     var total = 0;
+    // clear weekends from data
+    var weekday = [];
+    var i = 0;
+    Object.values(data).forEach(value => {
+      if (moment(value.event_date).isoWeekday() < 6) {
+      } else {
+        weekday.push(i)
+      }
+      i++;
+    });
+    for (var idx = weekday.length -1; idx >= 0; idx--)
+    data.splice(weekday[idx],1);
+
```

Comment:
Sorry for my lack of understanding, but what does this do?

Reply:
splice will remove those matching indices from the object.

Reply:
so we are looping through the "indexes" that we matched previously (for the dates that are weekends) and pushed to wekend array through weekend.push. then we are using those indexes to identify what to remove from the data Object.

Reply:
Not sure I understand, but OK 😼